config/settings/base.py

```diff
@@ -420,6 +420,12 @@ def GETKEY(group, request):
 # FILE_UPLOAD_REGION = env("FILE_UPLOAD_REGION", default="care-patient-staging")
 FILE_UPLOAD_KEY = env("FILE_UPLOAD_KEY", default="")
 FILE_UPLOAD_SECRET = env("FILE_UPLOAD_SECRET", default="")
+FILE_UPLOAD_BUCKET_ENDPOINT = env("FILE_UPLOAD_BUCKET_ENDPOINT", default=f"https://{FILE_UPLOAD_BUCKET}.s3.amazonaws.com")
```

Comment:
Is this being used somewhere?

Reply:
it will be used for config that which bucket to use

```
def get_client_config():
    config = {
        BucketType.PATIENT.value: {
            "region_name": settings.CLOUD_REGION,
            "aws_access_key_id": settings.FILE_UPLOAD_KEY,
            "aws_secret_access_key": settings.FILE_UPLOAD_SECRET,
            "endpoint_url": settings.FILE_UPLOAD_BUCKET_ENDPOINT,
        },
        BucketType.FACILITY.value: {
            "region_name": settings.CLOUD_REGION,
            "aws_access_key_id": settings.FACILITY_S3_KEY,
            "aws_secret_access_key": settings.FACILITY_S3_SECRET,
            "endpoint_url": settings.FACILITY_S3_BUCKET_ENDPOINT,
        },
    }

    if settings.CLOUD_PROVIDER == CSProvider.GCP.value:
        config["endpoint_url"] = "https://storage.googleapis.com"

    return config
```